client/src/views/Chat.vue

```diff
@@ -329,6 +328,17 @@ export default {
 
 <!-- Add "scoped" attribute to limit CSS to this component only -->
 <style scoped>
+
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+*{
+  font-family: "Poppins", "Helvetica", sans-serif;
+}
+
+strong:hover{
+  cursor: pointer;
+}
```

Comment:
This hover behaviour is not limited to userId but all bold elements, You can add this to #userId as well.

Reply:
Done.